Makefile

```diff
@@ -7,7 +7,12 @@ DOCKER_RUN = docker run -it --rm -u $(UID):$(GID)
 .PHONY: build, deploy, docker-build
 
 build: lint
-	$(DOCKER_RUN) -v $(PWD):/home/akeneo/pim-docs/data $(DOCKER_IMAGE) sphinx-build -b html /home/akeneo/pim-docs/data /home/akeneo/pim-docs/data/pim-docs-build
+	$(DOCKER_RUN) -v $(PWD):/home/akeneo/pim-docs/data $(DOCKER_IMAGE) sphinx-build -nWT -b html /home/akeneo/pim-docs/data /home/akeneo/pim-docs/data/pim-docs-build
+		# Flags used here, not in `make html`:
```

Comment:
I would put it before the line, not after, but :+1: to document it

Reply:
OUKI

Reply:
fixed